docs/index.md

```diff
@@ -1,97 +1,114 @@
 
+   ```yaml
+   #labs
+      - product: "labs"
+        component:     # GitHub repository name without "labs" prefix, e.g. `GithubUsername`
+   ```
      
-           #labs
-               - product: "labs"
-                 component:     #githubreponame without "labs" prefix
-   
 #### Deploy application
 
 1. We practise [GitOps](https://www.weave.works/technologies/gitops/) for application deployment to Kubernetes.
 
-   CFT: Read the documentation and follow the instructions [here]( https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md)
+   Follow the app deployment [guide](hmcts/cnp-flux-config@master/docs/app-deployment-v2.md#add-a-new-application) in cnp-flux-config.
 
 #### Access application
 
 1. Access the deployed application using the URL.
 
+   ```
+   http://<product>-<component>-sbox.service.core-compute-sandbox.internal 
+   ```  
    
-      http://<product>-<component>-sbox.service.core-compute-sandbox.internal   
-   
-#### Customize application
+#### Customise application
 
-1. Helm Charts can be customised by updating corresponding values file for each environment. Values files located under `/charts/<repo-name>/values.<ENV>.template.yaml`  
- 
-2. Environment variables can be passed by updating the corresponding values file in Helm chart. 
+1. We are going to customise the application by changing default landing page for the application by passing environment variables. 
+
+2. Helm Charts can be customised by updating `values.yaml` file located under `/charts/<repo-name>` directory.  
+
+3. Environment variables can be passed by updating values file in the Helm chart. 
  
-       
-       java:
-         environment:
-           FAVOURITE_FRUIT: plum   # KEY must be in uppercase
+   ```yaml
+   java:
+     environment:
+       FAVOURITE_FRUIT: plum   # KEY must be in uppercase
+   ```
+4. Update code to reference environment variable in [file](src/main/java/uk/gov/hmcts/reform/mohanalatest/controllers/RootController.java).
 
-#### Troubleshooting
+   ```java
+    public ResponseEntity<String> welcome() {
+        return ok("Welcome to labs-<githubusername>, my favourite fruit is " +  System.getenv("FAVOURITE_FRUIT"));
+    }
+    ```
+5. Review and merge `Pull Request`.
 
-1. Troubleshooting
+6. Run the jenkins pipeline against the `master` branch.
 
-     [Follow](https://hmcts.github.io/ways-of-working/troubleshooting/#troubleshooting-issues) the link to refer to troubleshoot steps.
-     - Pod/Application logs can viewed using `kubectl` command.
-     - For Deployment issue check Flux.  
-        
+7. Access the application using same `URL`.
+
+
+## Troubleshooting
+
+See our [troubleshooting](https://hmcts.github.io/ways-of-working/troubleshooting/#troubleshooting-issues) guide.
         
 
 ## Slack Channels
 
 - `#golden-path` is for support requests to the Platform Operations team
```

Comment:
it is not for support requests, it's for community discussion about the tutorials